lighthouse-core/audits/time-to-interactive.js

```diff
@@ -78,11 +78,17 @@ class TTIMetric extends Audit {
       const fMPts = timings.fMPfull + timings.navStart;
 
       // look at speedline results for 85% starting at FMP
-      const eightyFivePctVC = artifacts.Speedline.frames.find(frame => {
+      let eightyFivePctVC = artifacts.Speedline.frames.find(frame => {
         return frame.getTimeStamp() >= fMPts && frame.getProgress() >= 85;
       });
+
+      // Check to avoid closure compiler null dereferencing errors
+      if (eightyFivePctVC === undefined) {
+        eightyFivePctVC = 0;
+      }
```

Comment:
This is probably the ickiest change made. I tried a few different combinations of branching out to check for `undefined` after the speedline frame find/visual ready timing, moving that `getTimeStamp()` around etc but kept running into null dereferencing errors. If there's a cleaner way to fix the build than this, happy to apply.